src/scale/Log.ts

```diff
@@ -36,9 +36,28 @@ const roundingErrorFix = numberUtil.round;
 const mathFloor = Math.floor;
 const mathCeil = Math.ceil;
 const mathPow = Math.pow;
-
 const mathLog = Math.log;
 
+/**
+ * symmetric log, allowing negative values for logarithmic scales
+ */
+function symMathLog(x: number): number {
+    if (x >= 0) {
+        return mathLog(x);
+    }
```

Comment:
This review seems stale/abandoned ,but on of the off chance that someone is paying attention,

I think if x == 0, this should return 0. Math.log(0) will return -Infinity, and I don't think that's helpful for people with data that can range from large negative/positive numbers, or zero.

See: https://github.com/apache/echarts/issues/17459 

Proposed:
```
if (x === 0) {
  return 0;
}
return x > 0 ? mathLog(x) : -mathLog(-x);
```

Reply:
Good catch! I've updated the PR. Thank you!